seed.js

```diff
@@ -3,69 +3,62 @@
 
 var db = require('./models');
 
-var characters = [
+var characterList = [
 	{
 		name: "Spider-Man",
-		Abilities: ["Web-Slinger", "Swingin'", "Thwip, Thwip, Pow!"], 
+		abilities: ["Web-Slinger", "Swingin'", "Thwip, Thwip, Pow!"], 
 		image: "/images/Spider-Man-Free-Download-PNG.png"
 	}, {
 		name: "Iron-Man",
-		Abilities: ["Repulsor Ray", "Uni-Beam", "Billion-Dollar Punch"], 
+		abilities: ["Repulsor Ray", "Uni-Beam", "Billion-Dollar Punch"], 
 		image: "/images/Iron-Man-PNG-Image.png"
 	}, {
 		name: "Captain-America",
-		Abilities: ["Shield Bash", "Shield Toss", "American Justice"], 
+		abilities: ["Shield Bash", "Shield Toss", "American Justice"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"
 	}, {
 		name: "Dead-Pool",
-		Abilities: ["Spinning Groin Punch", "Spinny Sword Attack", "Hold Up"], 
+		abilities: ["Spinning Groin Punch", "Spinny Sword Attack", "Hold Up"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"
 	}, {
 		name: "Ultron",
-		Abilities: ["Encephalo-Ray", "Eradication"], 
+		abilities: ["Encephalo-Ray", "Eradication"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"
 	}, {
 		name: "Black-Panther",
-		Abilities: ["Roundhouse Tactical Strikes", "Panther Strike", "For Wakanda!"], 
+		abilities: ["Roundhouse Tactical Strikes", "Panther Strike", "For Wakanda!"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Guillotine",
-		Abilities: ["La Force d'Ancetres", "Coup de Massue", "Coup de Grace"], 
+		abilities: ["La Force d'Ancetres", "Coup de Massue", "Coup de Grace"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Storm",
-		Abilities: ["Lightning Strike", "Gale", "Tempest"], 
+		abilities: ["Lightning Strike", "Gale", "Tempest"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Winter-Soldier",
-		Abilities: ["Deuces are wild", "Fist of the Red Army"], 
+		abilities: ["Deuces are wild", "Fist of the Red Army"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Groot",
-		Abilities: ["Grow limbs into Earth", "I am Groot"], 
+		abilities: ["Grow limbs into Earth", "I am Groot"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "DareDevil",
-		Abilities: ["Stick Strike", "Flash Kick", "Deliverance"], 
+		abilities: ["Stick Strike", "Flash Kick", "Deliverance"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}
 ];
 
+db.Character.remove({}, function(err, characters){
+	db.Character.create(characterList, function(err, createdCharacters){
+	  if (err){
+	    console.log("Error:", err);
+	    return;
+	  }
 
-
-// db.Character.remove({}, function(err, characters){
-  
-
-// 	db.Character.create(characters, function(err, characters){
```

Comment:
This line was not working, because you named both the callback function argument `characters` and your hardcoded data `characters`. 

This `db.Character.create` is trying to create the `characters` on line 56, which is not what you want.

I renamed the hardcoded data `charactersList` and referenced that variable here on line 56.